glib/src/auto/functions.rs

```diff
@@ -750,6 +902,13 @@ pub fn unlink(filename: impl AsRef<std::path::Path>) -> i32 {
     unsafe { ffi::g_unlink(filename.as_ref().to_glib_none().0) }
 }
 
+#[doc(alias = "g_unsetenv")]
+pub fn unsetenv(variable: impl AsRef<std::path::Path>) {
```

Comment:
why does this takes a Path ? lol

Reply:
That needs to stay manual then, I guess. Should be an `OsStr`. Or can this also be configured in `Gir.toml`?

Reply:
Yes, see my other comment. `string_type = "os_string"`